web_advanced_search_x2x/static/src/js/web_advanced_search_x2x.js

```diff
@@ -37,10 +37,6 @@ odoo.define('web_advanced_search_x2x.search_filters', function (require) {
             this.operators.push({
                 'value': 'domain', 'text': core._lt('is in selection'),
             });
-            // Avoid hiding filter when using special widgets
-            this.events["click"] = function (event) {
```

Comment:
Yikes! Such a stupidity I did!

The problem happens because, like in Python, JS objects are always passed by reference. In this case it means we are changing the `events` object that is bound here, which comes directly from the class, not the instance. If that class is `Widget`, you can imagine the problem surface...

So the fix is to create a new object with all needed properties and bind it to the instance, try this instead of removing these lines:

```js
this.events = $.extend({}, this.events, {
    events: function (event) {
        event.stopPropagation();
    },
});
```

Reply:
Thanks for this fix. It works. But there is still other problems we should fix. I will investigate when I got time.